hedera-node/src/main/java/com/hedera/services/store/contracts/StaticEntityAccess.java

```diff
@@ -222,6 +282,83 @@ public long supplyOf(final TokenID tokenId) {
         return token.totalSupply();
     }
 
+    /**
+     * Returns the accountsKycGrantedByDefault flag of the given token.
+     *
+     * @param tokenId the token of interest
+     * @return the token's accountsKycGrantedByDefault flag
+     */
+    public boolean accountsKycGrantedByDefault(final TokenID tokenId) {
```

Comment:
All of these fields that look up the token, grab a single field, and then let it all fall away should be grouped together into one token info method.